src/main/java/org/folio/circulation/domain/LoanRepository.java

```diff
@@ -75,6 +75,15 @@ public LoanRepository(Clients clients) {
       .thenApply(mapResult(loanAndRelatedRecords::withLoan));
   }
 
+  public CompletableFuture<Result<Loan>> updateLoanItemInStorage(Loan loan) {
+    if (loan == null || loan.getItem() == null) {
+      return completedFuture(of(() -> null));
+    }
+
+    itemRepository.updateItem(loan.getItem());
+    return completedFuture(succeeded(loan));
```

Comment:
```suggestion
    return itemRepository.updateItem(loan.getItem())
        .thenApply(r -> succededResult(loan));
```

@mrdpopov, this should resolve the timeout issues, that you've faced in tests.

Reply:
It did, thanks